lib/l10nUtil.js

```diff
@@ -149,4 +158,7 @@ const defaultReplacements = [
   [/(?<!authenticate )people(?! with slow connections?)/g, 'profiles'],
   [/(Person)(?!\w)/g, 'Profile'],
   [/(person)(?!\w)/g, 'profile'],
+  [/Bookmarks Bar\n/g, 'Bookmarks\n'],
```

Comment:
nit: couldn't this be done in 1 line by using case-insensitive regex  `/bookmarks bar\n/gi`?

Reply:
no because the third one is replaced by `bookmarks`